pkg/compose/watch.go

```diff
@@ -419,6 +419,10 @@ func (t tarDockerClient) Exec(ctx context.Context, containerID string, cmd []str
 	return nil
 }
 
+func (t tarDockerClient) Untar(ctx context.Context, id string, archive io.ReadCloser) error {
+	return t.s.apiClient().CopyToContainer(ctx, id, "/", archive, moby.CopyToContainerOptions{})
```

Comment:
Should we set `CopyUIDGID: true`?

Engine uses that to make the files owned by the container user if set:
https://github.com/moby/moby/blob/4aa1bea04af697a4b6b6dbe5b6bef90747a3596d/daemon/archive_tarcopyoptions_unix.go